src/Constants.ts

```diff
@@ -51,6 +51,16 @@ export class Constants {
      */
     public static readonly JCR_CONTENT = 'jcr:content';
 
+    /**
+     * Query param for aem mode flag name.
+     */
+    public static readonly AEM_MODE = 'aemmode';
+
+    /**
+     * AEM mode: edit
+     */
+    public static readonly AEM_EDIT = 'edit';
```

Comment:
What about `AEM_MODE_EDIT`?

Reply:
Let's align and extend the naming convention agreed in

https://github.com/adobe/aem-spa-page-model-manager/pull/11/commits/f892e3c077400d551334cbfc34e73b5821eb165d#r484457657

Reply:
+1 `AEM_MODE_EDIT`